source/process/guest-article.rst

```diff
@@ -0,0 +1,21 @@
+Guest Articles for Mattermost Apps and Services
```

Comment:
Should it be `Guest Blog Posts` since we call them blog posts in the .com website?

Also `for Mattermost Apps and Services` is inconsistent with our other pages. E.g. `Case Studies` doesn't have it.